spring-cloud-gcp-data-datastore/src/main/java/org/springframework/cloud/gcp/data/datastore/repository/query/PartTreeDatastoreQuery.java

```diff
@@ -99,15 +107,17 @@
 	 * @param datastoreTemplate used to execute the given query.
 	 * @param datastoreMappingContext used to provide metadata for mapping results to objects.
 	 * @param entityType the result domain type.
+	 * @param projectionFactory the projection factory.
```

Comment:
Maybe add an additional few words describing what the purpose of projection factory/what its used for here?